Compass/LoginViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  LoginViewController.swift
+//  Compass
+//
+//  Created by Csabi on 08/10/2018.
+//  Copyright © 2018 Csabi. All rights reserved.
+//
+
+import UIKit
+import Parse
+
+class LoginViewController: UIViewController {
+    
+    var isloginPage = true
+    var activityIndicator = UIActivityIndicatorView()
+    
+    let titleLabel = UILabel(frame: CGRect.zero)
+    let NametextField = UITextField(frame: CGRect.zero)
```

Comment:
Generally you should always make properties start with a lower case, so I'd be tempted to rename this to `nameTextField`